src/apps/main/watchface_digital.cpp

```diff
@@ -90,8 +101,9 @@ void drawTime24Hour(OswHal* hal) {
 
 void drawSteps(OswHal* hal) {
   uint32_t steps = hal->getStepCount();
+  int width = numPositiveDigits(steps);
   hal->gfx()->setTextSize(2);
-  hal->gfx()->setTextCursor(((220/2)-defaultFontXOffset(1, 2)), 210 - defaultFontYOffset(1, 2) / 2);
+  hal->gfx()->setTextCursor(120 - hal->gfx()->getTextOfsetColumns(width/2), 210 - defaultFontYOffset(1, 2) / 2);
```

Comment:
better: `setTextCenterAligned()`, then we don't need the numPositiveDigits.

Reply:
Updated to use setTextCenterAligned and tested working great